docs/reference/transform/limitations.asciidoc

```diff
@@ -33,6 +33,14 @@ upgrade from 7.2 to a newer version, and {transforms} have been created in 7.2,
 the {transforms} UI (earler {dataframe} UI) will not work. Please wait until all 
 nodes have been upgraded to the newer version before using the {transforms} UI.
 
+[float]
+[[transform-rolling-upgrade-limitation]]
+==== {transforms-cap} Transforms reassignment suspended during a rolling upgrade from 7.2 and 7.3
+
+If your cluster contains mixed version nodes, for example during a rolling
+upgrade from 7.2, 7.3 to a newer version, transforms whose nodes are stopped will
```

Comment:
```suggestion
upgrade from 7.2 and 7.3 to a newer version, transforms whose nodes are stopped will
```

Reply:
In a rolling upgrade there are only 2 versions involved, "from" is either 7.2 _or_ 7.3 and "to" is 7.4, 7.5, 7.6...

Sounds to me like "or" would be the right choice.